lib/order.rb

```diff
@@ -0,0 +1,40 @@
+class Order
+  attr_reader :current_items
+
+  def initialize
+    @current_items = []
+  end
+
+  def select_items(menu)
+    loop do
+      puts 'Choose a dish to add to your order'
+      selection = gets.chomp
```

Comment:
Ah, user input, my old friend

Reply:
It's not fully working yet - fine in irb, but the mocking of stdin is an absolute nightmare (or at least I've made it one)